lib/ffe.gd

```diff
@@ -30,15 +31,19 @@
 ##  <P/>
 ##  &GAP; can represent elements of all finite fields
 ##  <C>GF(<A>p^d</A>)</C> such that either
-##  (1) <A>p^d</A> <M>&lt;= 65536</M> (in which case an extremely efficient
-##      internal representation is used);
+##  (1) <A>p^d</A> <M>&lt;=</M> <C>MAXSIZE_GF_INTERNAL</C> (in which case an 
+##      extremely efficient internal representation is used);
 ##  (2) d = 1, (in which case, for large <A>p</A>, the field is represented
 ##      using the machinery of residue class rings
 ##      (see section&nbsp;<Ref Sect="Residue Class Rings"/>) or
 ##  (3) if the Conway polynomial of degree <A>d</A> over the field with
 ##      <A>p</A> elements is known, or can be computed
 ##      (see <Ref Oper="ConwayPolynomial"/>).
 ##  <P/>
+##
+##  <C>MAXSIZE_GF_INTERNAL</C> may depend on the word size of your computer
+##  and the version of &GAP; but will typically be either <M>2^{16}</M> or
+##  <M>2^{24}</M>.<P/>
```

Comment:
Maybe insert another "empty" line (i.e., `##`) after this, to make the paragraph break more visible if one reads the comment directly, not via GAPDoc?